R/configurations.R

```diff
@@ -81,3 +81,46 @@ set_timestamp_format <- function(ts_format = "%Y-%m-%dT%H:%M:%S%z", confirm = TR
 get_timestamp_format <- function() {
   .config$ts_format
 }
+
+#' Set Log File Rotation
+#'
+#' Set configuration for the number of lines after which log files are.
+#' truncated. The function will echo out the currently set line number if
+#' is set, or else it will echo that there is no limit set.
+#'
+#' This function provides a means of setting automatic log file truncation.
+#' Log files are not truncated when the value is set to `NULL`, which is
+#' also the default setting.
+#'
+#' @param rotate_lines The number of log entries to keep in the logfile.
+#'   Defaults to 100,000, see `rotate_logs()`.
+#' @param confirm Print confirmation message of timestamp format? Defaults to
+#'   `TRUE`.
+#'
+#' @examples
+#' set_rotate_lines(100)  # set limit to 100 lines
+#' set_rotate_lines(NULL) # turn off auto-truncation
+#'
+#' @export
+set_rotate_lines <- function(rotate_lines = 100000, confirm = TRUE) {
```

Comment:
I think given the naming convention of the function itself (`set_rotate_lines()` vs. e.g. `set_rotation_on()`), this should default to `NULL` instead of an integer, since it expects an integer. Might need a check in `rotate_logs()` as well if `NULL` is passed in, so it doesn't crash.